lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -4,6 +4,11 @@ all_vnis:
   config_get: "show vni"
   config_get_token: '/^(\d+)\s/'
 
+all_vnis_n9k:
```

Comment:
Don't use a platform-specific tag, just update the existing `all_vnis` tag on line 3 to differentiate:

`
all_vnis:
  /N9K/:
    multiple: true
    config_get: "show running vlan"
    config_get_token: '/vn-segment (\d+)/'
  else:
    config_get: "show vni"
    config_get_token: '/^(\d+)\s/'